pkg/client/cache/index.go

```diff
@@ -14,7 +16,40 @@ const (
 // ImageStreamReferenceIndexFunc is a default index function that indexes based on image stream references.
 func ImageStreamReferenceIndexFunc(obj interface{}) ([]string, error) {
 	switch t := obj.(type) {
-	// TODO: Add support for build configs
+	case *buildapi.BuildConfig:
+		var keys []string
+
```

Comment:
i'll delete the one in the deploymentconfig case just below this as well....  though personally i like a newline separation between variable declarations and code logic.